log_bucket/variables.tf

```diff
@@ -0,0 +1,18 @@
+variable "cdn_logs_path" {
+  description = "Prefix for CloudFront logs"
+  default     = "cdn/"
+}
+
+variable "alb_logs_path" {
+  description = "Prefix for ALB logs"
+  default     = "alb/"
+}
+
+variable "s3_logs_path" {
+  description = "Prefix for S3 access logs"
+  default     = "s3/"
+}
+
+variable "bucket" {
```

Comment:
since this needs to be unique, i'd suggest you set it inside here instead of relying on the caller do it.

In other situations we used domain name + environment name + some identifying suffix like `adhoc.team-shared-logs` or something. But the details are up to you